### PR TITLE
Chat Styling [Mentor]

### PR DESCRIPTION
**Pull request by itsomkathe**
The styling of Mentor section has been done as per the Figma file for both the versions.
All the CSS is not converted to MUI as some selectors of SCSS are not present in MUI.
The ones which were possible, have been converted.

### CODE REVIEW

File: src/pages/Mentor/ChatNameBar/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            style={{ width: "4px", height: "4px" }}
+            className={classes.chatDot}
+          />
+        )}
+        <Typography
+          style={{
+            fontWeight: 400,
+            color: "#6D6D6D",
+            fontSize: !desktop && "14px",
+            marginTop: mobile && "6px",
```
Comment (kartiks26) [inline src/pages/Mentor/ChatNameBar/index.js]: can here the sx md property will work??

File: src/pages/Mentor/Message/index.js [hunk 2]
```diff
@@ -112,23 +129,19 @@ export default ({
         }`}
       >
         {!isSelf && senderName && (
-          <Avatar name={senderName} style={{ marginRight: 12 }} />
+          <Avatar
+            name={senderName}
+            style={{ height: "40px", width: "40px", marginRight: 12 }}
```
Comment (kartiks26) [inline src/pages/Mentor/Message/index.js:139]: Dont use much of Inline styles

File: src/pages/Mentor/Messages/index.js [hunk 3]
```diff
@@ -23,45 +24,47 @@ export default ({
...
-              let shouldDisplayDayMarker = false;
-              if (messages[index - 1]) {
-                shouldDisplayDayMarker = getAreDatesOnDifferentDays(
-                  messages[index - 1].origin_server_ts,
-                  message.origin_server_ts
+    <>
+      <Container className="messages">
+        <div className="messages-content" onScroll={onScroll} ref={messagesRef}>
+          {messages &&
+            messages
```
Comment (kartiks26) [inline src/pages/Mentor/Messages/index.js]: Make habbit of using ?. syntax here for array destructure  or the filter thing its more of cleaner approch

File: src/pages/Mentor/Messages/index.js [hunk 4]
```diff
@@ -23,45 +24,47 @@ export default ({
...
+              .filter((message) => message.type === "m.room.message")
+              .map((message, index) => {
+                let shouldDisplayDayMarker = false;
+                if (messages[index - 1]) {
+                  shouldDisplayDayMarker = getAreDatesOnDifferentDays(
+                    messages[index - 1].origin_server_ts,
```
Comment (kartiks26) [inline src/pages/Mentor/Messages/index.js]: For here as well better use  messages?.[index - 1]?.origin_server_ts, so it will never crash

**Review by kartiks26 — COMMENTED** [review]
Some comments

File: src/pages/Mentor/ChatNameBar/index.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+  const [roomName, setRoomName] = React.useState("");
+  const classes = useStyles({ desktop, laptop, mobile });
+
+  React.useEffect(() => {
+    rooms.map((room) => {
+      if (room.roomId === selectedRoomId) {
+        setRoomName(room.name);
+      }
+    });
```
Comment (kartiks26) [inline src/pages/Mentor/ChatNameBar/index.js:22]: here as well try using ?. syntax if it happens the name is null from backend then we might get crash issues

File: src/pages/Mentor/Messages/styles.scss [hunk 6]
```diff
@@ -14,19 +14,60 @@
...
   display: flex;
   flex-direction: column;
 }
 
 .messages-day-marker {
-  background-color: rgba(163, 163, 163, 0.1);
-  font-weight: bold;
-  font-size: 13px;
+  font-weight: 400;
+  color: #6d6d6d;
```
Comment (kartiks26) [inline src/pages/Mentor/Messages/styles.scss:29]: use colors form constant file which is used as in other files

File: src/pages/Mentor/Messages/styles.scss [hunk 7]
```diff
@@ -14,19 +14,60 @@
...
+    align-items: center;
+  }
+  .chat-name {
+    font-size: 18px;
+  }
+  .dot {
+    color: #6d6d6d;
+    width: 4px;
+    height: 4px;
+    margin: 0 8px 0 8px;
```
Comment (kartiks26) [inline src/pages/Mentor/Messages/styles.scss]: `margin : 0 8px;`
Is same right

File: src/pages/Mentor/Messages/styles.scss [hunk 8]
```diff
@@ -14,19 +14,60 @@
...
   border-radius: 4px;
   padding: 8px;
 }
+
+.chat-name-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 59px;
+  padding: 16px 32px 16px 32px;
```
Comment (kartiks26) [inline src/pages/Mentor/Messages/styles.scss]: `padding :16px 32px`
Is same right

File: src/pages/Mentor/RoomNav/styles.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+      padding: "16px 32px",
+      cursor: "pointer",
+      display: "flex",
+      justifyContent: "space-between !important",
+      backgroundColor: "#E9F5E9",
+    },
+    title: {
+      fontSize: "16px !important",
+      color: "#2E2E2E",
+      lineHeight: "24px !important",
```
Comment (kartiks26) [inline src/pages/Mentor/RoomNav/styles.js]: Why we need !important here ??

File: src/pages/Mentor/styles.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles(
+  () => ({
+    batchGroupsContainer: {
+      boxSizing: "border-box",
+      height: "59px",
+      padding: "16px 16px 16px 16px",
```
Comment (kartiks26) [inline src/pages/Mentor/styles.js]: `padding :16px` might work

**Review by kartiks26 — COMMENTED** [review]
part 2

File: src/pages/Mentor/Message/index.js (hunk 2 above)
Comment (itsomkathe) [reply src/pages/Mentor/Message/index.js:139]: For this if we do not add inline styles it does not apply. We'll see some solution for it.

File: src/pages/Mentor/Messages/index.js (hunk 3 above)
Comment (itsomkathe) [reply src/pages/Mentor/Messages/index.js]: It was not done by me. But we'll fix this.

File: src/pages/Mentor/ChatNameBar/index.js (hunk 5 above)
Comment (itsomkathe) [reply src/pages/Mentor/ChatNameBar/index.js:22]: Okay

File: src/pages/Mentor/ChatNameBar/index.js (hunk 1 above)
Comment (itsomkathe) [reply src/pages/Mentor/ChatNameBar/index.js]: For this feature, max 768px has been used for mobile.

File: src/pages/Mentor/RoomNav/styles.js (hunk 9 above)
Comment (itsomkathe) [reply src/pages/Mentor/RoomNav/styles.js]: Before adding prefix it was required. I've removed it now

Comment (kartiks26) [reply src/pages/Mentor/RoomNav/styles.js]: @itsomkathe resolve one which you have worked on

File: src/pages/Mentor/Message/index.js (hunk 2 above)
Comment (kartiks26) [reply src/pages/Mentor/Message/index.js:139]: @itsomkathe no issues

File: src/pages/Mentor/ChatNameBar/index.js (hunk 1 above)
Comment (kartiks26) [reply src/pages/Mentor/ChatNameBar/index.js]: @Poonam-Singh-Bagh jason did tell us something for this right to use it in different way

**Comment by kartiks26** [comment]
@Poonam-Singh-Bagh Once you merge dev to main please consider this merging

**Pull request closed without merging**